_posts/2017-05-03-impactkit.md

```diff
@@ -6,10 +6,108 @@ categories: archive impactkit
 indico: https://indico.cern.ch/event/624551/
 ---
 
-The Impact Kit is a three-day course in which participants follow advanced
-lectures and get a chance to obtain some hands-on experience on the LHCb
-software. Projects are proposed, and in addition participants are encouraged to
-bring their own problems to the table.
+The second LHCb Impactkit workshop was held at CERN from 3rd May and 5th May 2017, with 19 participants from 13 institutions across 5 countries. 
```

Comment:
I partly disagree, and changed it to:
from 3rd to 5th May 2017
Waiting for a native speaker to comment!